tools/wakucanary/wakucanary.nim

```diff
@@ -0,0 +1,118 @@
+import
+  std/[strutils, sequtils, tables],
+  confutils,
+  chronos,
+  stew/shims/net
+import
+  libp2p/protocols/ping,
+  libp2p/crypto/[crypto, secp]
+import
+  ../../waku/v2/node/peer_manager/peer_manager,
+  ../../waku/v2/utils/peers,
+  ../../waku/v2/node/wakunode2,
+  ../../waku/v2/node/waku_payload,
+  ../../waku/v2/utils/peers
+
+# protocols and their tag
+const ProtocolsTable = {
+  "store": "/vac/waku/store/",
+  "static": "/vac/waku/relay/",
```

Comment:
Why are we calling `relay` protocol `static` instead of just... `relay`?

Reply:
I got the `static` from #754. @jm-clius wdyt?

Reply:
That's just a legacy artifact of how the `status-go` canary was written. No reason to cargo-cult stuff like that.

Reply:
Indeed. We still refer to `staticnode` which happens to be relay nodes currently (but could be any statically-configured node in future). I'd also be keen to move away from this terminology in future, so better to stick with the protocol names (as you have done - just confirming that I agree.)